cliquet/resource.py

```diff
@@ -652,7 +652,8 @@ def _next_page_url(self, sorting, limit, last_record):
         params['_token'] = token
 
         service = current_service(self.request)
-        next_page_url = self.request.route_url(service.name, _query=params)
+        next_page_url = self.request.route_url(service.name, _query=params,
+                                               **self.request.matchdict)
```

Comment:
I really don't know how to unit test this :( Any hints ?

Before : tests broken. After : tests green.